# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/koffice/hotfix.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ur koffice-1.2.95/kchart/kdchart/KDChartAxisParams.cpp koffice-1.2.95-fixed/kchart/kdchart/KDChartAxisParams.cpp
--- koffice-1.2.95/kchart/kdchart/KDChartAxisParams.cpp	2003-12-16 11:02:34.000000000 +0100
+++ koffice-1.2.95-fixed/kchart/kdchart/KDChartAxisParams.cpp	2003-12-28 18:33:51.000000000 +0100
@@ -31,6 +31,7 @@
 #include "KDChartAxisParams.moc"
 #endif
 
+#include <limits.h>
 
 /**
   \class KDChartAxisParams KDChartAxisParams.h
--- koffice-1.2.95/kchart/kdchart/KDDrawText.h	2003-12-16 11:02:38.000000000 +0100
+++ koffice-1.2.95-fixed/kchart/kdchart/KDDrawText.h	2003-12-28 18:32:43.000000000 +0100
@@ -29,13 +29,14 @@
 #ifndef __KDDRAWTEXT_H__
 #define __KDDRAWTEXT_H__
 
+#include <limits.h>
+
 #include <qpainter.h>
 #ifdef linux
 #include <values.h>
 #endif
 #include "KDChartGlobal.h"
 
-
 struct KDDrawTextRegionAndTrueRect
 {
     QRegion region;
